ui/store/articles.js

```diff
@@ -16,6 +16,8 @@ export const state = () => {
 // =================================================
 export const getters = {
   articles: (state) => state.articles,
+  todayArticles: (state) => state.articles.filter((article, index) => index < 2),
```

Comment:
@swilgosz Currently backend doesn't return dates. I "mocked" 2 first articles to be marked as "today" articles temporarily.